e2e/cypress/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -0,0 +1,348 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @ldap
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+function setLDAPTestSettings(config) {
+    return {
+        siteName: config.TeamSettings.SiteName,
+        siteUrl: config.ServiceSettings.SiteURL,
+        teamName: '',
+        user: null,
+    };
+}
+
+// assumes the CYPRESS_* variables are set
+// assumes that E20 license is uploaded
+// for setup with AWS: Follow the instructions mentioned in the mattermost/platform-private/config/ldap-test-setup.txt file
+context('ldap', () => {
+    let testChannel;
+    let testTeam;
+    let testUser;
+
+    describe('LDAP Group Sync Automated Tests', () => {
+        beforeEach(() => {
+            // * Check if server has license for LDAP
+            cy.apiRequireLicenseForFeature('LDAP');
+
+            // # Initial api setup
+            cy.apiInitSetup().then(({team, user}) => {
+                testTeam = team;
+                testUser = user;
+
+                // # Update LDAP settings
+                cy.apiGetConfig().then(({config}) => {
+                    setLDAPTestSettings(config);
+                });
+
+                // # Login as sysadmin and add board-one to test team
+                cy.apiAdminLogin();
```

Comment:
I'm seeing redundant/unneeded admin login. Maybe move this at the start of `beforeEach` and then remove all `cy.apiAdminLogin()` at the start of each `it` block?

Reply:
Oh snap! I didn't even realise that my last login is as admin. Thank for pointing it out. Fixed 👍